aries-site/src/pages/components/select.js

```diff
@@ -20,13 +29,178 @@ const Select = () => {
         <Subsection name={title} level={1} topic={topic}>
           <SubsectionText>{page.description}</SubsectionText>
           <Example
-            code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/input/SelectExample.js"
+            code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/select/SelectExample.js"
+            designer="https://designer.grommet.io/select?id=HPE-design-system-matthew-glissmann-hpe-com"
             docs="https://v2.grommet.io/select?theme=hpe#props"
+            figma="https://www.figma.com/file/fZJdRcNrvZ1JjdPRYHpFFU/hpe-select?node-id=117%3A4"
           >
             <SelectExample />
           </Example>
         </Subsection>
       </ContentSection>
+      <ContentSection>
+        <Subsection name="Guidance">
+          <SubsectionText>
+            Select is a flexible input element able to serve a wide variety of
+            use cases. Though flexible, asking "is Select the best tool in this
+            circumstance?" is a healthy exercise. Variants of Select, such
+            as&nbsp;
+            <Anchor href="#select-with-search">Select with Search</Anchor>, or
+            alternatives like{' '}
+            <Anchor href="/components/radiobuttongroup">
+              RadioButtonGroup
+            </Anchor>{' '}
+            or a{' '}
+            <Anchor href="/components/textinput#textinput-with-suggestions">
+              TextInput with suggestions
+            </Anchor>{' '}
+            may be more appropriate for crafting an elegant user experience.
+          </SubsectionText>
+        </Subsection>
+        <Subsection name="When to use Select" level={3} gap="small">
+          <SubsectionText>
+            Because Select is flexible and familiar, the following questions are
+            helpful to consider alternatives and/or confirm that Select is the
+            most appropriate element:
+          </SubsectionText>
+          <Box width={{ max: 'large' }}>
+            <BulletedList
+              level={3}
+              items={[
+                `Is there a need to conserve screen space? Select may be 
+              appropriate.`,
+                `Ensuring collection of standardized values? Select may 
+              be appropriate.`,
+                <Text size="small">
+                  Are there less than five options? Consider a CheckBoxGroup or
```

Comment:
Would menu also be an option for less then 5 options ?

Reply:
Menu as an alternative to Select?

Reply:
ignore that brain not 100% lol